webapp/tests/karma/ts/modals/edit-user/update-password.component.spec.ts

```diff
@@ -37,16 +34,11 @@ describe('UpdatePasswordComponent', () => {
         }
       )
     };
-    languagesService = {
-      get: sinon.stub().resolves(
-        [{code: 'en', name: 'English'}]
-      )
-    };
-    translateService = { get: sinon.stub().returns(new Observable()) };
     return TestBed
       .configureTestingModule({
         imports: [
           FormsModule,
+          TranslateModule.forRoot({ loader: { provide: TranslateLoader, useClass: TranslateFakeLoader } }),
```

Comment:
These tests started failing because testbed didn't recognize the translate pipe.